llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -444,8 +444,8 @@ def FeatureStdExtV
 def HasVInstructions    : Predicate<"Subtarget->hasVInstructions()">,
       AssemblerPredicate<
           (any_of FeatureStdExtZve32x),
-          "'V' (Vector Extension for Application Processors), 'Zve32x' or "
```

Comment:
Doesn't V also depend on Zve64d, and so V->Zve64d->Zve64f->Zve32f->Zve32x? So we could just simplify this to Zve32x

Reply:
Technically yes, but people working in the application processor space might pretend that Zve doesn't exist.